Lesson22/Merve/script.js

```diff
@@ -0,0 +1,51 @@
+const form = document.getElementById('toDoForm');
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+    event.preventDefault();
+    const input = document.getElementById('toDoInput');
+    if (input.value.trim() === '') {
+        alert('Please enter a todo item.');
```

Comment:
Nice that you added input validation 👍
For the error message, you can add an empty element in HTML (e.g. `<p id="errorMsg"></p>`). If the input value is not as expected, you can set the text of the <p> element to the error message:
`document.getElementById('errorMsg').innerText = 'Please enter a todo item.';`.

Then, on the next form submit, you can re-set the value of the 'errorMsg' element to empty string (`document.getElementById('errorMsg').innerText = '';`)